docs/core/docker/intro-net-docker.md

```diff
@@ -160,43 +160,42 @@ The [dotnet-framework:4.6.2 sample](https://github.com/Microsoft/dotnet-framewor
 
 * [Operations Management Suite (OMS) Linux agent overview](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/docs/Docker-Instructions.md#overview)
 
-* [Operations Management Suite (OMS) images on DockerHub](https://hub.docker.com/r/microsoft/vsts-agent/) 
+* [Operations Management Suite (OMS) images on DockerHub](https://hub.docker.com/r/microsoft/vsts-agent/)
 
 * [Operations Management Suite (OMS) images on GitHub](https://github.com/Microsoft/OMS-docker)
 
 #### Microsoft Azure Command Line Interface (CLI)
 
-* [Microsoft Azure Command Line Interface (CLI) images on DockerHub](Docker image for Microsoft Azure Command Line Interface) 
+* [Microsoft Azure Command-Line Interface (CLI) images on DockerHub](https://hub.docker.com/r/microsoft/azure-cli/)
```

Comment:
@JRAlexander can you make sure this is the link you wanna point to?

Reply:
Thought I replied a while back. Sorry!